frontend/src/metabase/lib/notifications.ts

```diff
@@ -280,7 +292,15 @@ export const formatNotificationCheckSchedule = ({
       }
       break;
     }
+    case "cron":
+      try {
+        return t`Check ${getScheduleExplanation(cronSchedule)}`;
+      } catch {
+        return null;
+      }
```

Comment:
Since getScheduleExplanation already wraps the call to cronstrue in try/catch you might not need this try/catch block, I think

Reply:
Maybe we could put the try/catch in `explainCronExpression`?

Reply:
Good catch (pun intended :D). I think leaving try/catch at `getScheduleExplanation` level is better just to handle a potential case of `explainCronExpression` returning some invalid value. I'm not sure if that's possible, but I'd feel more protected that way :)